exporters/geneva/example/stress_test_linux.cc

```diff
@@ -28,9 +28,11 @@
 #include <chrono>
 #include <sys/times.h>
 
+#ifdef HAVE_BOOST
```

Comment:
Is boost optional, and missing it doesn't break the build?

Reply:
yeah it's optional. We use some of it's functionality if present, else use that from STL.